docs/using/creating-gists.md

```diff
@@ -0,0 +1,26 @@
+# Creating gists
+
+GitHub for Visual Studio enables easy creation of gists directly from the Visual Studio Editor.
```

Comment:
The "easy" here doesn't sound quite right to me. Maybe remove the "easy" or change to "With GitHub for Visual Studio you can easily create gists directly from the editor" or something?